client/layout/masterbar/gravatar.scss

```diff
@@ -4,6 +4,7 @@
 .gravatar-masterbar {
 	position: absolute;
 	padding: 18px 20px;
+	z-index: 1000;
```

Comment:
It might be worth looking into the `z-index()` sass function to handle this

Reply:
@noahtallen Thank you for the great tip, I have updated it :)